metrics/web/lib/feature_config/domain/entities/feature_config.dart

```diff
@@ -11,15 +11,20 @@ class FeatureConfig extends Equatable {
   /// Indicates whether the debug menu feature is enabled.
   final bool isDebugMenuEnabled;
 
+  /// Indicates whether the public dashboard feature is enabled.
```

Comment:
🎯 
```suggestion
  /// A flag that indicates whether the public dashboard feature is enabled.
```
See [Effective dart](https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases) for more details.